src/binder/bind/bind_ddl.cpp

```diff
@@ -24,40 +22,94 @@ namespace binder {
         }
     }
-    return make_unique<BoundCreateNodeClause>(tableName, std::move(boundProperties), primaryKeyIdx);
+    auto boundExtraInfo = std::make_unique<BoundNodeExtraCreateTableInfo>(primaryKeyIdx);
+    auto boundCreateInfo = std::make_unique<BoundCreateTableInfo>(
+        tableName, std::move(boundProperties), std::move(boundExtraInfo));
+    return make_unique<BoundCreateTable>(
+        StatementType::CREATE_NODE_TABLE, tableName, std::move(boundCreateInfo));
 }
 
 std::unique_ptr<BoundStatement> Binder::bindCreateRelTableClause(
     const parser::Statement& statement) {
-    auto& createRelClause = (CreateRelClause&)statement;
-    auto tableName = createRelClause.getTableName();
+    auto& createTableClause = (CreateTableClause&)statement;
+    auto tableName = createTableClause.getTableName();
     if (catalog.getReadOnlyVersion()->containTable(tableName)) {
         throw BinderException("Rel " + tableName + " already exists.");
     }
-    auto boundProperties = bindProperties(createRelClause.getPropertyNameDataTypes());
+    auto createInfo = createTableClause.getInfo();
+    auto boundProperties = bindProperties(createInfo->propertyNameDataTypes);
     for (auto& boundProperty : boundProperties) {
         if (boundProperty->getDataType()->getLogicalTypeID() == LogicalTypeID::SERIAL) {
             throw BinderException("Serial property is not supported in rel table.");
         }
     }
-    auto relMultiplicity = getRelMultiplicityFromString(createRelClause.getRelMultiplicity());
-    return make_unique<BoundCreateRelClause>(tableName, std::move(boundProperties), relMultiplicity,
-        bindNodeTableID(createRelClause.getSrcTableName()),
-        bindNodeTableID(createRelClause.getDstTableName()));
+    auto extraInfo = (RelExtraCreateTableInfo*)createInfo->extraInfo.get();
+    auto srcTableID = bindNodeTableID(extraInfo->srcTableName);
+    auto srcTableSchema =
+        (NodeTableSchema*)catalog.getReadOnlyVersion()->getTableSchema(srcTableID);
+    auto srcPkDataType = srcTableSchema->getPrimaryKey()->getDataType();
+    auto dstTableID = bindNodeTableID(extraInfo->dstTableName);
+    auto dstTableSchema =
+        (NodeTableSchema*)catalog.getReadOnlyVersion()->getTableSchema(dstTableID);
+    auto dstPkDataType = dstTableSchema->getPrimaryKey()->getDataType();
+    auto relMultiplicity = getRelMultiplicityFromString(extraInfo->relMultiplicity);
+    auto boundExtraInfo = std::make_unique<BoundRelExtraCreateTableInfo>(
+        relMultiplicity, srcTableID, dstTableID, srcPkDataType->copy(), dstPkDataType->copy());
+    auto boundCreateInfo = std::make_unique<BoundCreateTableInfo>(
+        tableName, std::move(boundProperties), std::move(boundExtraInfo));
+    return make_unique<BoundCreateTable>(
+        StatementType::CREATE_REL_TABLE, tableName, std::move(boundCreateInfo));
+}
+
+static constexpr char RDF_IRI[] = "_IRI";
```

Comment:
Move them to InternalKeyWord?

Reply:
Let's keep it here and see if it's needed for querying. If not, we don't need to put in InternalKeyword